test/tools/Modules/PSSysLog/PSSysLog.psm1

```diff
@@ -789,12 +805,13 @@ function Get-PSOsLog
 
     if ($TotalCount -eq 0)
     {
-        Get-Content @contentParms | ConvertFrom-OsLog -After $After -Id $Id
+        Get-Content @contentParms | Where-Object {![string]::IsNullOrEmpty($_)} | ConvertFrom-OsLog -After $After -Id $Id
     }
     else
     {
         [string] $filter = [string]::Format("com.microsoft.powershell.{0}: (", $id)
-        Get-Content @contentParms -filter {$_.Contains($filter)} | ConvertFrom-OsLog -Id $Id -After $After | Select-Object -First $maxItems
+        Write-Warning "this code path `Get-PSOsLog -TotalCount` should not be used if the message field is needed!"
```

Comment:
I suggest filing an issue against this.  The intent is for the entire log entry to be returned even when using TotalCount.